packages/smooth_app/lib/pages/onboarding/welcome_page.dart

```diff
@@ -12,84 +12,61 @@ class WelcomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final Size screenSize = MediaQuery.of(context).size;
-    // Side padding is 8% of total width.
-    final double sidePadding = screenSize.width * 0.08;
-    // Top padding is 16% of total width.
-    final double topPadding = screenSize.height * 0.016;
-    // Bottom padding is 4% of total width.
-    final double bottomPadding = screenSize.height * 0.04;
     final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
     final TextStyle headlineStyle =
         Theme.of(context).textTheme.headline2!.apply(color: Colors.white);
-    final TextStyle largeButtonTextStyle =
-        Theme.of(context).textTheme.headline3!.apply(color: Colors.white);
     final TextStyle bodyTextStyle =
         Theme.of(context).textTheme.bodyText1!.apply(color: Colors.white);
+    // Side padding is 8% of total width.
+    double sidePadding(double screenWidth) => screenWidth * 0.08;
```

Comment:
You could put it directly in a `final`: `final double sidePadding = screenSize.width * .08;` (instead of creating a method).

Reply:
Done